src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import PeopleTable from '../PeopleTable/PeopleTable';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsloading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isLoading, setIsloading] = useState(false);
  const [isError, setIsError] = useState(false);
```
it's redundant to specify types for primitives

Reply:
fixed